test/repl/proxysql_repl_tests/bin/debezium-check.bash

```
@@ -0,0 +1,74 @@
+  echo "topic: $topic, partition: $partition, offset: $offset, gtid: $gtid"
+
+  # extract the last number from gtid and update max_gtid if necessary
+  gtid_num=$(echo "$gtid" | cut -d':' -f2)
+  if (( gtid_num > ${max_gtid#*:} )); then
+    max_gtid="$gtid"
+  fi
+
+done
+
+# extract the last number from gtid_executed
+gtid_executed=$(mysql ${SSLOPT} -h${MYSQL1_HOST}${INFRA} -P${MYSQL1_PORT} -uroot -proot -s -N -e "SHOW GLOBAL VARIABLES LIKE 'gtid_executed';" 2>/dev/null | cut -d ':' -f 2- | cut -d '-' -f 2)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)